layouts/default.vue

```diff
@@ -548,15 +552,48 @@ export default {
             <span>{{ t('nav.clusterTools') }}</span>
           </a>
         </n-link>
-        <div class="version text-muted">
-          {{ displayVersion }}
+        <div v-if="showProductFooter" class="footer">
```

Comment:
When embedded in the dashboard and not stand alone i think this should remain as per other 'cluster' pages (only shows the rancher version)

Reply:
yep, I added a layer to determine isSingleVirtualCluster。

Done.

Reply:
I think I made this mistake before and got muddled again this time. The footer should match the cluster explorer as much as possible, though in order to show the custom Harvester support page it needs to keep the support link. So I was wrong to suggest that only the version should show.

Reply:
@richard-cox  Indeed, we should show the support page, but we should not show the switch language.